### DISCUSSION TITLE
Enable optional display of issue title with new syntax #123+ (with a plus sign at the end)

### DISCUSSION THREAD

**Discussion by fgnievinski** [community/community#41837] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

there are pros and cons for the automatic display of issue titles. 

recently it's been updated so that it's automatically displayed in lists, but not everyone liked it:
- #4321

the proposed new syntax `#123+` (with a plus sign at the end) would give more control for users to opt in when and where to activate the display of issue titles.

I've created this community discussion so that it can collect votes; a full description of the proposal has been presented in more detail elsewhere:
- https://github.com/isaacs/github/issues/1020

**Comment by Goldbishop** [comment]
2-cents

I agree with most that the raw syntax `#4231` is great for mass Issue Releases.

But there are circumstances unrelated to Milestones and Releases that the Title expansion (with or without the number) is useful without a bulleted list requirement.

The proposed Opt-In `#4231+` is a sound option for those that need it.  We can't control how users may abuse it but having the feature is worth more than the mitigation thought of not having it.

It also seems the ToDoList `- [ ]`, in a Release Comment (ref: https://github.com/Goldbishop/OneBigBag/releases/tag/20230827_U36), feature in GitHub is not considered a Bulleted List and does not have the same behavior as it does in Issue comments.  In the aforemented reference, I want to leverage the ToDoList (regardless of its functionality) for effect and the Issue Title is relevant with a link to.

**Comment by Kludex** [comment]
This would be great.

**Reply by zyv** [reply]
This is a very annoying feature indeed, but what's worse is that I'm not sure of a good way to suppress it. I would really like it if it were opt-in, but unfortunately that's not the case.

**Comment by zyv** [comment]
A duplicate:

* #43484
